pkg/config/templates/rancherd-17-loki-stack.yaml

```diff
@@ -0,0 +1,24 @@
+resources:
+  - apiVersion: management.cattle.io/v3
+    kind: ManagedChart
+    metadata:
+      name: loki-stack
+      namespace: fleet-local
+    spec:
+      chart: loki-stack
+      releaseName: loki-stack
+      #      we might want to add this to the cos config eventually, so we can change the version with ldflags
+      version: 2.7.0
+      defaultNamespace: cattle-logging-system
+      repoName: harvester-charts
+      targets:
+        - clusterName: local
+          clusterSelector:
+            matchExpressions:
+              - key: provisioning.cattle.io/unmanaged-system-agent
+                operator: DoesNotExist
+      values:
```

Comment:
Will check if grafana, promtail default values are false, when false, will remove them from managedchart to avoid mis-usage.